packages/components/src/components/menu-flyout-list/menu-flyout-list.tsx

```diff
@@ -288,14 +290,22 @@ export class MenuFlyoutList {
 
   setPosition() {
     const { top, left } = this.triggerRect;
-    this.hostElement.style.top = `${top}px`;
-    this.hostElement.style.left = `${left}px`;
+    this.hostElement.style.left = `${left - 4}px`;
+    if (this.trigger().tagName === 'SCALE-TELEKOM-NAV-ITEM') {
+      this.hostElement.style.top = `${top - 12}px`;
+      this.hostElement.style.left = `${left - 24}px`;
```

Comment:
I wish there was a more generic way to do this instead of hard-coding (imagine using CSS variables for this instead of assigning directly in JavaScript and then having an offset variable defaulting to 0 on each axis), but we've done similar things in other parts of this same component, I guess it's fine… 🤓 (nothing to do here, but just wanted to point it out)

Reply:
true, I was trying to get this change in quickly for the release, but I guess now there is a bit more time I could use spacing-tokens here